bibliopixel/util/duration.py

```diff
@@ -0,0 +1,106 @@
+import re
+
+"""Convert a string duration to a number of seconds.
+"""
+
+SI_PREFIXES = (
+    (10 ** 24, 'yotta', 'Y'),
+    (10 ** 21, 'zetta', 'Z'),
```

Comment:
REALLY? So we run an animation for 1Zs and it's still going at the heat death of the universe...
Also, the ones at the other end of the time scale. These are completely useless for any real world usage. I get the desire for completion but this is just kinda silly.